packages/app/src/app/pages/WorkspaceFlows/SignUpUBB.tsx

```diff
@@ -46,11 +43,6 @@ export const SignUpUBB = () => {
     );
   }
 
-  if ((workspaceId && ubbBeta === true) || isAdmin === false) {
-    // Page was accessed with a non-ubb workspace id
-    return <Redirect to={dashboardUrls.recent(workspaceId)} />;
```

Comment:
Is this an intentional change?

Reply:
Yes, but the admin part is still needed, need to revert